src/main/java/org/zone/commands/structure/zone/flags/greetings/GreetingsMessageViewCommand.java

```diff
@@ -0,0 +1,56 @@
+package org.zone.commands.structure.zone.flags.greetings;
+
+import net.kyori.adventure.text.Component;
+import org.spongepowered.api.command.CommandResult;
+import org.zone.commands.system.ArgumentCommand;
+import org.zone.commands.system.CommandArgument;
+import org.zone.commands.system.NotEnoughArgumentsException;
+import org.zone.commands.system.arguments.operation.ExactArgument;
+import org.zone.commands.system.arguments.zone.ZoneArgument;
+import org.zone.commands.system.context.CommandContext;
+import org.zone.region.Zone;
+import org.zone.region.flag.FlagTypes;
+import org.zone.region.flag.greetings.GreetingsFlag;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+public class GreetingsMessageViewCommand implements ArgumentCommand {
+    public static final ExactArgument ZONE = new ExactArgument("zone");
+    public static final ExactArgument FLAGS = new ExactArgument("flags");
```

Comment:
Could you make the id ``flag`` to make it consistant

Reply:
wdym

Reply:
You see where it says ``"flags"`` could this be ``"flag"``?

Reply:
done will update